components/ingest-service/grpc/grpc.go

```diff
@@ -57,24 +60,33 @@ func Spawn(opts *serveropts.Opts) error {
 	}
 
 	// Register Status Server
-	ingestStatus := server.NewIngestStatus(client, A1migration)
+	ingestStatus := server.NewIngestStatus(client, migrationer)
 	ingest.RegisterIngestStatusServer(grpcServer, ingestStatus)
 
 	// Initialize elasticsearch indices or trigger a migration
 	//
 	// This initialization will happen inside a goroutine so that we can then
 	// report the health of the system. We might have to do the same for the
 	// migration tasks.
-	migrationNeeded, err := A1migration.MigrationNeeded()
+	migrationNeeded, err := migrationer.MigrationNeeded()
 	if err != nil {
+		log.WithFields(log.Fields{"error": err}).Fatal("Failed checking for migration")
```

Comment:
nit: Sine all these logs are just adding an error as a field, I would suggest:
```suggestion
		log.WithError(err).Fatal("Failed checking for migration")
```